CHANGELOG.md

```diff
@@ -15,16 +15,18 @@ Versioning](http://semver.org/spec/v2.0.0.html).
  
 ### Changed
 * "About" page now sources meaningful content from `aboutPageURL` (#755)
+* z-index values adjusted from highest (101) to lowest (51) stated values to play nicer with Angular Material (#760)
```

Comment:
How impactful is this change? Is this going to feel like a breaking change to upgrading framework apps? What will app developers typically need to do in updating apps to a framework version with this change?

Reply:
It shouldn't make much of an impact. I would guess most downstream devs would be writing custom z-indices to make content sit on top of our top bar or other high-index content, in which case they would be unaffected. The only case in which they might be affected is if they have z-indices for content sitting between the top bar and some other content (like the menu), which seems unlikely.